internal/oci/runtime_vm.go

```diff
@@ -243,15 +243,14 @@ func (r *runtimeVM) StartContainer(c *Container) error {
 
 	// Spawn a goroutine waiting for the container to terminate. Once it
 	// happens, the container status is retrieved to be updated.
-	var err error
 	go func() {
-		_, err = r.wait(r.ctx, c.ID(), "")
+		_, err := r.wait(r.ctx, c.ID(), "")
```

Comment:
I feel like we shouldn't just swallow this error...

Reply:
I added log for non-nil error